python/ray/experimental/serve/examples/echo_slo_reverse.py

```diff
@@ -50,7 +59,7 @@ def echo_v1(flask_request, response="hello from python!"):
     print("> [REMOTE] Pinging handle.remote(response='{}',slo_ms={})".format(
         response, slo_ms))
     # slo can be specified via remote function
-    f = handle.remote(response=response, slo_ms=slo_ms)
+    f = handle.options(relative_slo_ms=slo_ms).remote(response=response)
```

Comment:
This is testing the slo override right? if so can you add a comment above if not can you test that case?